features/element-hiding.json

```diff
@@ -675,15 +675,6 @@
                     }
                 ]
             },
-            {
-                "domain": "dallasnews.com",
-                "rules": [
-                    {
-                        "selector": "[data-targeting]",
-                        "type": "hide-empty"
-                    }
-                ]
-            },
```

Comment:
Not sure why this was here to begin with, this rule doesn't do anything since we already have a generic `[data-targeting]` rule.